scripts/fingerprint_snapshot.py

```diff
@@ -0,0 +1,35 @@
+from mp_time_split.core import MPTimeSplit
+
+from matbench_genmetrics.utils.match import (
+    cdvae_cov_comp_fingerprints,
+    cdvae_cov_struct_fingerprints,
+)
+
+mpt = MPTimeSplit(target="energy_above_hull")
+
+mpt.load(dummy=True)
+
+original_inputs = mpt.inputs
+
+comp_fingerprints = cdvae_cov_comp_fingerprints(mpt.inputs)
+struct_fingerprints = cdvae_cov_struct_fingerprints(mpt.inputs)
+
+# save the fingerprints and upload to figshare
```

Comment:
https://pypi.org/project/pystow/ might be useful for this

Reply:
Hadn't heard of this one. Looking into it. Thanks!

Reply:
also https://github.com/cthoyt/zenodo-client/! but obv this is for zenodo instead of figshare. This package completely automates the update and versioning cycle for zenodo

Reply:
@cthoyt thanks! I'll need to check out hosting datasets via Zenodo

Reply:
Yeah, using GitHub for hosting big datasets (like bigger than 100mb) isn’t so great since you go into LFS territory

Reply:
@cthoyt agreed, I've typically been using FigShare for datasets over ~10 MB, and strongly avoid LFS (which of course has its uses). I've been using Zenodo to get DOIs for my GitHub repos (i.e. DOI the code) but hadn't thought of using it directly for dataset files. Zenodo seems really nice - large dataset limits, code formatting support, etc. I think I'll try it out next time. Aside: [maybe a comparable package for FigShare](https://github.com/cognoma/figshare)